samples/openthread/openthread.conf

```diff
@@ -0,0 +1,48 @@
+#
```

Comment:
@carlescufi Is it ok to have a dangling project file in the `openthread` top directory? I think it's quite unprecenteded. Personally, I'd see it in `samples/openthread/common/`, but we've already gone through this before, where it was rejected to have non-sample directory in the samples tree. I have no other idea where to keep it though.

As a reference, they have such a directory in Zephyr `samples/net` tree (but there nothing useful at the moment):
https://github.com/zephyrproject-rtos/zephyr/tree/master/samples/net/common

Reply:
I kind of dislike this placement, but I do understand the purpose of sharing common settings.

But as a minimum, this file should be named `overlay-openthread.conf`, as to show this is an overlay file.
Similar to `zephyr/samples/net`:
https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/net/sockets/echo_server/overlay-tls.conf

Will be considering a bit more on this proposal.

Reply:
I see that this has now been moved to `samples/openthread/common`. I am fine with that.